olive/engine/engine.py

```diff
@@ -436,13 +436,17 @@ def run_accelerator(
     def setup_passes(self, accelerator_spec: AcceleratorSpec):
         # clean the passes
         self.passes.clear()
-        for config in self.pass_config.values():
+        for name, config in self.pass_config.items():
             pass_cls: Type[Pass] = config["type"]
             pass_cfg = config["config"]
             pass_cfg = pass_cls.generate_search_space(accelerator_spec, pass_cfg, config["disable_search"])
             p = pass_cls(accelerator_spec, pass_cfg, config["disable_search"])
             self.register_pass(
-                p, host=config["host"], evaluator_config=config["evaluator"], output_name=config["output_name"]
+                p,
+                host=config["host"],
+                evaluator_config=config["evaluator"],
+                output_name=config["output_name"],
+                name=name,
```

Comment:
>name=name, [](http://example.com/codeflow?start=16&length=10)

the parameter name is after the instance. Move there to keep consistency

Reply:
Thanks, I just treat it as kw arguments then ignore the order.
Will wait the first around CI test results. Then to fix it.